app/forms/mappings/Mappings.scala

```diff
@@ -29,9 +29,12 @@ trait Mappings extends Formatters with Constraints {
 
 
   protected def localDate(
-                           invalidKey: String
+                           invalidKey: String,
```

Comment:
you can bring invalidiKey up a line and move the indentation here I think? Like how you have done for populateErrorMsg